plugins/inventory/proxmox.py

```diff
@@ -234,14 +241,21 @@ def _get_agent_network_interfaces(self, node, vmid, vmtype):
                 )
             )['result']
 
+            if "error" in ifaces and "class" in ifaces["error"] and (ifaces["error"]["class"] in ["Unsupported", "CommandDisabled"]):
+                # This happens on Windows, even though qemu agent is running, the IP address
+                # cannot be fetched, as it's unsupported, also a command disabled can happen.
+                return result
+
```

Comment:
```suggestion
            if "error" in ifaces:
                if "class" in ifaces["error"]:
                    # This happens on Windows, even though qemu agent is running, the IP address
                    # cannot be fetched, as it's unsupported
                    if ifaces["error"]["class"] in ["Unsupported", "CommandDisabled"]:
                        self.display.v(
                            "Retrieving network interfaces from guest agents on older Windows versions is not supported"
                        )
                return result

```
Looks like this should always return early if `"error"` is in the result object so this would handle all instances of that with additional log detail for the specific instances identified.

Reply:
it's not really older windows versions it's older qemu agents also.

Reply:
Ah, wasn't clear on that. Whatever text would help the end user understand why this fact is not populated with their guest configuration is best. The suggestion is really just some logging for this scenario.

Reply:
Done, I've separated the Unsupported and CommandDisabled